command-parser/src/parser.rs

```diff
@@ -115,7 +115,12 @@ impl<'a> Parser<'a> {
 
     fn find_command(&'a self, buf: &'a str) -> Option<&'a str> {
         self.config.commands().iter().find_map(|command| {
-            if buf.starts_with(&command[..]) {
+            if buf
+                .split_whitespace()
```

Comment:
Not sure if forcing commands to not have whitespace is what we want, but I figure it's fine for now. 

We'd have to do something more exotic to fix #80 otherwise.